### PR TITLE
systemd environment variables

### PR DESCRIPTION
**Pull request by skylerto**
**This is by no means a complete PR, just testing to see if there's any interest**

Signed-off-by: skylerl <skylerl@indellient.com>

### Description

Change adds support for adding environment variables to the docker daemon service script

### Issues Resolved

Addresses #859 

### Check List

- [ ] All tests pass. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/TESTING.MD>
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [ ] All commits have been signed for the Developer Certificate of Origin. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/CONTRIBUTING.MD>

### CODE REVIEW

**Comment by chasebolt** [comment]
sorry for the late update on this. if you rebase id be happy to merge this in!

File: libraries/docker_service_manager_systemd.rb [hunk 1]
```diff
@@ -19,6 +19,23 @@ class DockerServiceManagerSystemd < DockerServiceBase
     action :start do
       create_docker_wait_ready
 
+      # stock systemd unit file
```
Comment (chasebolt) [inline libraries/docker_service_manager_systemd.rb]: this shouldnt be an addition as this block was moved above the `.socket` template. you need to delete the duplicate copy on line 54.

Comment (skylerto) [inline libraries/docker_service_manager_systemd.rb]: good catch, thanks!

**Comment by skylerto** [comment]
Is there anything I can do to get those last tests passing? Looks like master is having the same issues.

**Comment by someara** [comment]
looks like there's problem with the chef-yum-docker cookbook... causing tests to explode. investigating

**Comment by chasebolt** [comment]
this is a two part issue. fedora-latest (26) doesnt support docker. and centos 7 is trying to install 17.05 which is considered experimental and not stable. i have a PR for these coming in.

**Comment by chasebolt** [comment]
can you rebase and squash your commits? travis is all fixed up now.

**Comment by skylerto** [comment]
I've been merging as opposed to rebasing, so all my commits are lost in the merge commits. :cry: could you provide some guidance on how I can fix this please?

**Comment by someara** [comment]
I did this manually here. 27fd84e1abc304494ebd17b55872abf5168ff13d
Thanks!

-s

**Comment by skylerto** [comment]
Thanks!

**Pull request closed without merging**